anax-in-k8s/Dockerfile.ubi.auto-upgrade-cron.ppc64el

```diff
@@ -19,7 +19,7 @@ RUN microdnf clean all \
   && microdnf install -y --nodocs --setopt=install_weak_deps=0 --disableplugin=subscription-manager ${REQUIRED_RPMS} \
   && microdnf clean all --disableplugin=subscription-manager \
   && rm -rf /mnt/rootfs/var/cache/* /mnt/rootfs/var/log/dnf* /mnt/rootfs/var/log/yum.* \
-  && curl -4LO https://storage.googleapis.com/kubernetes-release/release/`curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt`/bin/linux/ppc64le/kubectl \
+  && curl -4LO https://storage.googleapis.com/kubernetes-release/release/v1.33.0/bin/linux/amd64/kubectl \
```

Comment:
should this be `ppc64le`(or `ppc64el`, need your confirmation) in the path instead of `amd64`?